src/App.css

```diff
@@ -1,38 +1,745 @@
 .App {
   text-align: center;
 }
+:root {
+  --primaryColor: #1e282f;
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+  --secondaryColor: #487eb0;
+  --textColor: #dfe6e9;
 }
+body {
+  color: var(--textColor);
+}
+* {
+  padding: 0;
+  margin: 0;
+}
+input,
+button :focus {
+  outline: none;
+}
+ul {
+  list-style: none;
+}
+.link {
+  color: var(--textColor);
+  text-decoration: none;
+  cursor: pointer;
+}
+p {
+  margin: 0;
+}
+a {
+}
+.header {
+  position: fixed;
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+  width: 100%;
+  height: 10vh;
+  background-color: var(--primaryColor);
+  box-shadow: 0 5px 25px 0 rgb(0 0 0 / 30%);
+  z-index: 1;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.hamburger {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0rem 1rem;
+}
+.hamburger-icon {
+  font-size: 2.5rem;
+  color: var(--textColor);
+}
+.header-list-div {
+  width: 25%;
```

Comment:
Maintain consistency while defining unit sizes, if you are using `rem` use that only across your CSS files